src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -23,13 +25,44 @@
 
 NUM_DAYS = 28
 
+SEER_ANOMALY_DETECTION_CONNECTION_POOL = connection_from_url(
```

Comment:
Why do we add this here?

Reply:
oh oops, a relic of me trying to define it in 1 place rather than 5 but I ran into import errors and gave up, thinking I might try again in a separate PR